mapper/orm-coordination-outbox-polling/src/main/java/org/hibernate/search/mapper/orm/coordination/outboxpolling/cfg/impl/UuidDataTypeUtils.java

```diff
@@ -29,14 +30,19 @@ private UuidDataTypeUtils() {
 	 * <p>
 	 * Otherwise, when {@code value != "default" } a user passed value will be used.
```

Comment:
I was 50-50 on this one. I guess if we change the expectations and instead of a string, we say that a user can pass a SQL-type-code ... then it still can work. I'm not sure if there's a simple alternative though ... using something like `org.hibernate.annotations.Type` will still break this...

Reply:
From what I can see some configuration properties in Hibernate ORM accept either the name of a `SqlTypes` constant or its value, e.g. `hibernate.type.preferred_duration_jdbc_type`. Depending how it's implemented we might do the same here, automatically resolving a name to a code?

See https://github.com/hibernate/hibernate-orm/pull/6562/files#diff-fc943226b0861c9546796194526878ba1a19474f164a99ab16bb5cfc886f656d

Reply:
Hmmmm yeah. ORM uses this converter:
https://github.com/hibernate/hibernate-orm/blob/ae8b3f9a33f204086cffcb857a4863f20c2953ad/hibernate-core/src/main/java/org/hibernate/internal/util/config/ConfigurationHelper.java#L589-L611
while it is private and lives in an internal module the `JdbcTypeNameMapper` it is using is in a public module so I guess we can do something similar 👍🏻😃